src/JupyterContents.js

```diff
@@ -1,3 +1,8 @@
+// Copyright (c) Jupyter Development Team.
+// Distributed under the terms of the Modified BSD License.
```

Comment:
The license should be BSD-3-Clause according tho here: https://github.com/jupyterlab/jupyterlab/blob/master/packages/services/package.json#L18

Reply:
Ok, I copied this text from the current version in the upstream repo. And it's effectively the same license:
https://opensource.org/licenses/BSD-3-Clause

But I can change to be more explicit.

Reply:
```suggestion
// Distributed under the terms of the 3-Clause BSD License.
```

Reply:
I see, ok, didn't know, please feel free to choose one and merge, thanks!

Reply:
Let's keep it like this so we don't change what they wrote.